sql/moz-fx-data-shared-prod/telemetry_derived/newtab_desktop_daily_engagement_by_tile_id_position_v1/query.sql

```diff
@@ -0,0 +1,165 @@
+          AND pocket IS NULL
+          THEN 1
+        ELSE 0
+      END
+    ) AS impressions,
+    SUM(CASE WHEN click IS NOT NULL THEN 1 ELSE 0 END) AS clicks,
+    SUM(CASE WHEN pocket IS NOT NULL THEN 1 ELSE 0 END) AS pocketed,
+    SUM(CASE WHEN block IS NOT NULL THEN 1 ELSE 0 END) AS blocked
+  FROM
+    legacy_impression_data
+  CROSS JOIN
+    UNNEST(legacy_impression_data.tiles) AS flattened_tiles
+    WITH OFFSET AS alt_pos
+  GROUP BY
+    submission_timestamp,
+    client_id,
+    tile_id,
+    position
+),
+legacy_summary AS (
+  SELECT
+    submission_timestamp,
+    CAST(NULL AS STRING) AS recommendation_id,
+    tile_id,
+    position,
+    SUM(impressions) AS impression_count,
+    SUM(clicks) AS click_count,
+    SUM(pocketed) AS save_count,
+    SUM(blocked) AS dismiss_count
+  FROM
+    legacy_flattened_impression_data
+  WHERE
+    clicks < 3
+  GROUP BY
+    submission_timestamp,
+    tile_id,
+    position
+),
+  -- GLEAN Query
+glean_deduplicated_pings AS (
+  SELECT
+    submission_timestamp,
+    document_id,
+    normalized_country_code,
+    client_info,
+    events
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_live.newtab_v1`
+  WHERE
+    DATE(submission_timestamp) >= '2024-11-15'
+    AND DATE(submission_timestamp) = @submission_date
+  QUALIFY
+    ROW_NUMBER() OVER (
+      PARTITION BY
+        DATE(submission_timestamp),
+        document_id
+      ORDER BY
+        submission_timestamp DESC
+    ) = 1
+),
+glean_flattened_pocket_events AS (
+  SELECT
+    document_id,
+    submission_timestamp,
+    events.name AS event_name,
+    mozfun.map.get_key(events.extra, 'recommendation_id') AS recommendation_id,
+    mozfun.map.get_key(events.extra, 'tile_id') AS tile_id,
+    mozfun.map.get_key(events.extra, 'position') AS position,
+    COUNT(1) OVER (PARTITION BY document_id, events.name) AS user_event_count
+  FROM
+    glean_deduplicated_pings,
+    UNNEST(events) AS events
+  WHERE
+    events.category = 'pocket'
+    AND events.name IN ('impression', 'click', 'save', 'dismiss')
+    AND (
+      mozfun.map.get_key(events.extra, 'recommendation_id') IS NOT NULL
+      OR mozfun.map.get_key(events.extra, 'tile_id') IS NOT NULL
+    )
+    AND mozfun.map.get_key(events.extra, 'scheduled_corpus_item_id') IS NULL
+    AND SAFE_CAST(SPLIT(client_info.app_display_version, '.')[0] AS int64) >= 121
+),
+glean_summary AS (
+  SELECT
+    submission_timestamp,
+    recommendation_id,
+    COALESCE(SAFE_CAST(tile_id AS int), -1) AS tile_id,
+    COALESCE(SAFE_CAST(position AS int), -1) AS position,
+    SUM(CASE WHEN event_name = 'impression' THEN 1 ELSE 0 END) AS impression_count,
+    SUM(CASE WHEN event_name = 'click' THEN 1 ELSE 0 END) AS click_count,
+    SUM(CASE WHEN event_name = 'save' THEN 1 ELSE 0 END) AS save_count,
+    SUM(CASE WHEN event_name = 'dismiss' THEN 1 ELSE 0 END) AS dismiss_count
+  FROM
+    glean_flattened_pocket_events
+  WHERE
+    NOT (user_event_count > 50 AND event_name = 'click')
```

Comment:
Shouldn't we be removing all activity from suspicious clients, not just the click events?

```suggestion
    NOT (user_event_count > 50)
```